integration/bytes-as-base64/parameters.txt

```diff
@@ -0,0 +1 @@
+outputEncodeMethods=false,outputJsonMethods=true
```

Comment:
Pretty minor, but outputJsonMethods is already true by default, so maybe leave out?